included_software.md

```diff
@@ -9,19 +9,21 @@ For instructions on how to run this image locally to test your build, please see
 The specific patch versions included will depend on when the image was last built (except Ruby). It is highly suggested you depend only on minor versions, so that we can ensure the language has the latest updates (especially if security related).
 
 * Ruby - `RUBY_VERSION`, `.ruby-version`
-  * 2.6.2 (default)
-  * 2.7.2
+  * 2.7 (default)
+  * 2.6
+  * 3.0
   * Any version that `rvm` can install.
 * Node.js - `NODE_VERSION`, `.nvmrc`, `.node-version`
-  * 10 (default)
+  * 16.4 (default)
```

Comment:
Based on this [issue comment](https://github.com/netlify/build-image/issues/577#issuecomment-867988431), I think we're better off pinning Node to the major version (i.e. `16`) rather than to a minor.